File: mon-forum-back/src/main/java/com/forum/monforumback/entities/Message.java
package com.forum.monforumback.entities;

import java.time.LocalDateTime;

import org.springframework.data.annotation.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import org.springframework.data.mongodb.core.mapping.Document;

@Document
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Message {
	
	@Id
    private String idMessage;
	
	private String contentMessage;
	
	private int likesMessage;	
	
	private LocalDateTime createdDateMessage;
	
	private LocalDateTime updateDateMessage;
	
	private String idSujet;

}

File: mon-forum-client/src/app/models/message.ts
export class Messages {

    idMessage:string;
	
	contentMessage:string;
	
	likesMessage:number;	
	
	idSujet:string;

    createdDateMessage: Date;

    updateDateMessage: Date;

    constructor(idMessage:string, contentMessage:string, likesMessage:number, idSujet:string, createdDateMessage: Date,updateDateMessage: Date ){
            this.idMessage = idMessage;
            this.contentMessage = contentMessage;
            this.likesMessage = likesMessage;
            this.idSujet = idSujet;
            this.createdDateMessage = createdDateMessage;
            this.updateDateMessage = updateDateMessage;
    }
}